src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -363,6 +363,30 @@
         "wasm": "true"
       }
     },
+    "continuousIntegration": {
+      "displayName": "CI Templates",
```

Comment:
We should make CI conditional on selecting net7.0, to avoid us having to support CI pipelines for both net6 and 7
cc @dansiegel are you ok with this?

Reply:
No... it should not be conditional. The changes between the two should be fairly easy and we should be able to adopt some conditional logic in the template.json to switch values for the environment setup between net6/net7